src/binaryen.cpp

```diff
@@ -509,10 +509,12 @@ ExecutionResult BinaryenEngine::execute(
   return result;
 }
 
-void BinaryenEngine::loadModule(vector<uint8_t> const& code, wasm::Module & module)
+void BinaryenEngine::loadModule(bytes_view code, wasm::Module& module)
 {
   try {
-    wasm::WasmBinaryBuilder parser(module, reinterpret_cast<vector<char> const&>(code), false);
+    auto const b = reinterpret_cast<char const*>(code.begin());
+    auto const e = reinterpret_cast<char const*>(code.end());
+    wasm::WasmBinaryBuilder parser(module, {b, e}, false);
```

Comment:
These three lines are way less clear to me compared to the original one.

Reply:
At least vector is not casted.